src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -179,11 +179,15 @@ static void InsertContentView(UIScrollView platformScrollView, IScrollView scrol
 			var contentContainer = new ContentView()
 			{
 				View = scrollView.PresentedContent,
+#pragma warning disable CS0618 // Type or member is obsolete
 				CrossPlatformMeasure = ConstrainToScrollView(scrollView.CrossPlatformMeasure, platformScrollView, scrollView),
+#pragma warning restore CS0618
 				Tag = ContentPanelTag
 			};
 
+#pragma warning disable CS0618 // Type or member is obsolete
 			contentContainer.CrossPlatformArrange = ArrangeScrollViewContent(scrollView.CrossPlatformArrange, contentContainer, platformScrollView, scrollView);
+#pragma warning restore CS0618
```

Comment:
@hartez Is this something we need to consider for this PR? @jonathanpeppers will this still cause scroll views to live forever?

Reply:
Maybe we need a new way that allows for also attaching static methods? This way we can still use the delegates, but be static so they don't capture anything?

Reply:
@PureWeen what were your findings for using static methods? Did that help?

Reply:
But you can still create leaks with static methods, see the closures here:

https://github.com/dotnet/maui/blob/3b620952425ca3c4542c4632e05d09eb4f583e22/src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs#L195 https://github.com/dotnet/maui/blob/3b620952425ca3c4542c4632e05d09eb4f583e22/src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs#L227

Should we just avoid using *all* delegates? Otherwise we need to make it a build error if you use non-static ones -- or closures.